source/plugin/optional/usage.rst

```diff
@@ -205,7 +205,7 @@ just use
 .. code-block:: java
 
     getOptionalString()
-          .map(s -> s.toLowerCase)
+          .map(s -> s.toLowerCase())
```

Comment:
I'm not sure but maybe this "longish" version is here on purpose to show both variants. 
The next line is the short version.